### ISSUE TITLE
namingConvention fails

### ISSUE THREAD

**Issue by vtfn** [rossPatton/stylint#332]
using namingConventio = camelCase I got erros with varibles like this:
```stylus
$someColor = #fff
```
But no error is thrown to this
```stylus
someColor = #fff
```
Fails with code that should warn. Like this
```stylus
some-color = #fff
```
But complain about this
```stylus
$some-color = #fff
```

When set to ```lowercase-dash``` it doesn't warn with code a name like 
```stylus
someColor = #fff
```
But an error is thrown with
```stylus
$someColor = #fff
```
It also fails in placeholders

**Comment by rossPatton** [comment]
see https://github.com/rossPatton/stylint#namingconventionstrict--default-false-true--false-

**Comment by vtfn** [comment]
I don't see what you mean. Doesn't this option set whether or not to check other names than the variables?

I've tested it with ```namingConventionStrict``` both true and false and got the same problems.
All names in my code (except ids and classes, which is why I rather keep ```namingConventionStrict``` false) are in cammel case, and ```namingConvention``` is set to cammel, and I got the ```preferred naming convention is cammel``` error
